app/lib/BaseEditorController.php

```diff
@@ -699,7 +699,23 @@ public function Summary($pa_options=null) {
 		list($vn_subject_id, $t_subject) = $this->_initView($pa_options);
 		$vb_ajax_load_displays = $t_subject->getAppConfig()->get('enable_ajax_summary_displays');
 		session_write_close();
-		$this->SummaryData($pa_options, $vb_ajax_load_displays);
+		$this->SummaryInfo($pa_options, !$vb_ajax_load_displays || $this->request->isAjax());
+
+		if((defined('__CA_ENABLE_DEBUG_OUTPUT__') && __CA_ENABLE_DEBUG_OUTPUT__) || (bool)$this->request->config->get('display_template_debugger') || ($this->request->user->getPreference('show_template_debugger') !== 'hide')) {
```

Comment:
I think these make sense to place at the page load level, no need to replicate on every ajax request.

Reply:
Add Access check before rendering this.